owncloudData/src/test/java/com/owncloud/android/data/preferences/datasource/implementation/OCSharedPreferencesProviderTest.kt

```diff
@@ -0,0 +1,136 @@
+package com.owncloud.android.data.preferences.datasource.implementation
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.preference.PreferenceManager
+import com.owncloud.android.data.preferences.datasources.implementation.OCSharedPreferencesProvider
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.verify
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+class OCSharedPreferencesProviderTest {
+
+    private lateinit var ocSharedPreferencesProvider: OCSharedPreferencesProvider
+    private lateinit var context: Context
+    private lateinit var sharedPreferences: SharedPreferences
+    private lateinit var editor: SharedPreferences.Editor
+    val key = "test_key"
+
+    @Before
+    fun setUp() {
+        context = mockk(relaxed = true)
+        sharedPreferences = mockk(relaxed = true)
+        editor = mockk(relaxed = true)
+        mockkStatic(PreferenceManager::class)
+        every { PreferenceManager.getDefaultSharedPreferences(any()) } returns sharedPreferences
+        every { sharedPreferences.edit() } returns editor
+
+        ocSharedPreferencesProvider = OCSharedPreferencesProvider(context)
+    }
+
+    @Test
+    fun `put String - ok`() {
+        val value = "test_value"
+        ocSharedPreferencesProvider.putString(key, value)
+
+        verify (exactly = 1) { editor.putString(key, value).apply() }
```

Comment:
Forgot to mention in this class, but let's move the content of the `verify` block to a new line just as we did in the other test class 👍

Reply:
Done